plugins/modules/dcnm_interface.py

```diff
@@ -1214,6 +1339,184 @@
           admin_state: true                               # Flag to enable/disable Vlan interaface
           mode: vlan                                      # choose from [vlan, vlan_admin_state], default is "vlan"
 
+# AA FEX INTERFACES
+
+- name: Create AA FEX interfaces including optional parameters
```

Comment:
*nit:* Instead of AA can we use active-active in the name to be more descriptive?

Reply:
Sure, will change